debian/changelog

```diff
@@ -1,3 +1,49 @@
+cloud-init (21.2-0ubuntu1) hirsute; urgency=medium
```

Comment:
Our tooling uses the last changelog release, so gets it wrong for the first run in a new release:

```suggestion
cloud-init (21.2-0ubuntu1) impish; urgency=medium
```